README.md

```diff
@@ -477,6 +477,11 @@ Usage: `nerdctl start [OPTIONS] CONTAINER [CONTAINER...]`
 
 Unimplemented `docker start` flags: `--attach`, `--checkpoint`, `--checkpoint-dir`, `--detach-keys`, `--interactive`
 
+### :whale: nerdctl wait
```

Comment:
Could you run `doctor README.md` to sync the TOC

https://github.com/thlorenz/doctoc